bool data
uint8 axes
---
bool success
string message
